demisto_sdk/commands/common/hook_validations/description.py

```diff
@@ -67,7 +67,7 @@ def is_duplicate_description(self):
         if not re.match(PACKS_INTEGRATION_YML_REGEX, self.file_path, re.IGNORECASE):
             package_path = os.path.dirname(self.file_path)
             try:
-                md_file_path = glob.glob(os.path.join(os.path.dirname(self.file_path), '*_description.md'))[0]
+                md_file_path = glob.glob(os.path.join(self.file_path.split('.')[0], '_description.md'))[0]
```

Comment:
why not 
os.path.basename(self.file_path)

Reply:
I'm looking for the full path including file name without the extension  - I'll use: os.path.splitext instead